lib/cartopy/mpl/geoaxes.py

```diff
@@ -417,6 +417,34 @@ def coastlines(self, resolution='110m', color='black', **kwargs):
                                                       resolution, **kwargs)
         return self.add_feature(feature)
 
+    def tissot(self, rad_km=5e5, n_samples=80, lon_n=10, lat_n=6, **kwargs):
+        """
+        Adds Tissot's indicatrices to the current axes with the geodesic class.
+
+        Kwargs:
+
+            * rad_km - The radius in km of the the circles to be drawn.
+
+            * n_samples - Integer number of points sampled around the
+              circumference of each circle.
+
+            * lon_n - Number of circles in the longitudinal direction.
+
+            * lat_n - Number of circles in the latitudinal direction.
+
+        """
+        geod = cartopy.geodesic.Geodesic()
+        geoms = []
+
+        for lat in np.linspace(-80, 80, lat_n):
```

Comment:
It may be easiest to just provide an argument, but I think these limits should be determined by the extent of the map in some way. For example, say someone plots azimuthal equidistant centred on the North pole; they probably don't care too much about -80, but 85 or something would be visible.